Templates.sln

```diff
@@ -331,14 +331,37 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "GitHubCommenter-NodeJS", "G
 		Templates\GitHubCommenter-NodeJS\sample.dat = Templates\GitHubCommenter-NodeJS\sample.dat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-PowerShell-Modules", "HttpTrigger-PowerShell-Modules", "{BB1D67BC-8053-4658-B1AD-8A0AD0EB3AB8}"
+	ProjectSection(SolutionItems) = preProject
+		Templates\HttpTrigger-PowerShell-Modules\function.json = Templates\HttpTrigger-PowerShell-Modules\function.json
+		Templates\HttpTrigger-PowerShell-Modules\metadata.json = Templates\HttpTrigger-PowerShell-Modules\metadata.json
+		Templates\HttpTrigger-PowerShell-Modules\readme.md = Templates\HttpTrigger-PowerShell-Modules\readme.md
+		Templates\HttpTrigger-PowerShell-Modules\run.ps1 = Templates\HttpTrigger-PowerShell-Modules\run.ps1
+		Templates\HttpTrigger-PowerShell-Modules\sample.dat = Templates\HttpTrigger-PowerShell-Modules\sample.dat
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "modules", "modules", "{CE3B740F-A70F-4D44-8D0B-1DB8BA6DBA8B}"
+	ProjectSection(SolutionItems) = preProject
+		Templates\HttpTrigger-PowerShell-Modules\modules\Get-DateToday.psm1 = Templates\HttpTrigger-PowerShell-Modules\modules\Get-DateToday.psm1
+		Templates\HttpTrigger-PowerShell-Modules\modules\Show-Calendar.psd1 = Templates\HttpTrigger-PowerShell-Modules\modules\Show-Calendar.psd1

```

Comment:
Binary files won't be picked up by portal. Since portal tries to read every file as text when it tries to deploy templates.